nu_stats/bhmutils.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
```

Comment:
Nice! Could also expand this module to a class that handles the Stan interface (e.g. converts sim outputs into the correct data format, compiles and runs the Stan model, does some basic diagnostic events and classifies the events)